src/Components/Dialogs/GetPerson/getPerson.jsx

```diff
@@ -1,15 +1,18 @@
 import React from 'react';
 import { NavLink } from 'react-router-dom';
 
-import { Path } from '../../../core/constants';
-
 import style from './getPerson.module.css';
 
 const GetPerson = (props) => {
-  const { items } = props;
+  const { items, Path } = props;
   const itemsArray = Object.values(items);
 
-  const sd = ({ id, name }) => (<div key={id}><NavLink to={`${Path.dialog}/${id}`} className={style.item}>{name}</NavLink></div>);
+  const sd = ({ id, name, src }) => (
+    <div key={id} className={style.wrapper}>
+      <div className={style.imgContainer}><img src={src} className={style.img} alt="alt" onError="this.onerror=null;" /></div>
```

Comment:
Consider moving the onError handler into a separate dedicated function in the current component.

Reply:
Okey.i created